src/main/resources/application.yaml

```
@@ -44,6 +47,23 @@ spring:
       resourceserver:
         jwt:
           jwk-set-uri: http://idam-web-public/o/jwks
+      client:
+        registration:
+          idam-testing-support-api:
+            authorization-grant-type: client_credentials
+            client-id: idam-testing-support-api
+            client-secret: dummy
+            client-authentication-method: client_secret_post
+            redirect-uri: https://idam-testing-support-api
+            scope:
+              - profile
+              - create-active-user
+              - view-user
+              - delete-user
+        provider:
+          idam-testing-support-api:
+            token-uri: http://idam-api/o/token
```

http://idam-web-public/o/token
```suggestion
            token-uri: http://idam-api/o/token
```